src/components/normal-opening-hours-validity/NormalOpeningHoursValidity.tsx

```diff
@@ -72,6 +72,7 @@ const NormalOpeningHoursValidity = (): JSX.Element => {
                     openButtonAriaLabel="Valitse alkupäivämäärä"
                     required
                     value={startDateField.value}
+                    crossOrigin={undefined}
```

Comment:
What are these additions for? HDS documentation does not have these variables mentioned anywhere.

Reply:
Resolved with internal conversation, linked to https://github.com/creativetimofficial/material-tailwind/issues/427